packages/backend/src/functions/Project.ts

```diff
@@ -168,23 +168,25 @@ export default class Project {
     }
   }
 
-  private async removeSocket(socket: SocketIO.Socket): Promise<void> {
+  private removeSocket(socket: SocketIO.Socket): Promise<boolean> {
```

Comment:
When we are not async anymore we don't return a promise, just a boolean
```suggestion
  private removeSocket(socket: SocketIO.Socket): boolean {
```